purchase_order_line_seller_used/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module adds the Vendor Pricelist used in the Purchase Order Line every
+time it gets updated in the Odoo standard flow.
+
+In Odoo, every time the `Product Quantity` is changed, the new Vendor Pricelist
+is also searched. This Vendor Pricelist is then assigned to the Purchase Order
+Line to be then used as needed.
```

Comment:
Naive question : In which module / use case do you use this new field ?

Reply:
My need is for a custom module to restrict the PO confirmation if there is not a Vendor Pricelist used for any of the lines. But I have set it up here in this module as it can be used for other use cases, such as having the reverse relation, for example.
Perhaps, my use case could also be implemented as an OCA module. What do you think?

Reply:
>  Perhaps, my use case could also be implemented as an OCA module. What do you think?

Not a clear point of view. I personaly don't use this second feature. In fact, I don't understand why restrict confirmation in such situation.
But well, maybe it can be interesting for other OCA users.